gini/gini-vision-lib-ios
# Network plugin POC

In order to ease the integration of the Gini Vision Library, it has been integrated all the networking within the library. 
To make the networking optional, it has been split in different `subspecs`. (`Core`, `Networking` and `Networking+Pinning`).

### How to test
Run the example app and check that is working fine without any `GiniVisionDelegate` implementation.
Now replace the `GiniVision/Networking` with `GiniVision/Networking+Pinning`on your `Podfile` and check that the pinning is possible with the new methods.

### Merging
Not merge.
